cmake/onnxruntime_flatbuffers.cmake

```diff
@@ -28,3 +28,17 @@ if (NOT onnxruntime_BUILD_SHARED_LIB)
             RUNTIME   DESTINATION ${CMAKE_INSTALL_BINDIR}
             FRAMEWORK DESTINATION ${CMAKE_INSTALL_BINDIR})
 endif()
+
+if (GDK_PLATFORM)
+  # cstdlib only defines std::getenv when _CRT_USE_WINAPI_FAMILY_DESKTOP_APP is defined, which
+  # is probably an oversight for GDK/Xbox builds (::getenv exists and works).
```

Comment:
Do you have a contact to bug for this? (not that you're blocked, but just to fix it in the future)

Reply:
Yeah I have a few people I'm going to poke. :)